doc/managing_pachyderm/migrations.md

```diff
@@ -0,0 +1,291 @@
+# Migration 
+- [Introduction](#introduction)
+- [Note about 1.7 to 1.8 migrations](#note-about-1-7-to-1-8-migrations]
+- [General migration procedure](#general-migration-procedure)
+  - [Before you start: backups](#before-you-start-backups)
+  - [Migration steps](#migration-steps)
+    - [1. Pause all pipeline and data loading operations](#pause-all-pipeline-and-data-loading-operations)
+    - [2. Extract a pachyderm backup with the --no-objects flag](#extract-a-pachyderm-backup-with-the-no-objects-flag)
+    - [3. Clone your object store bucket](#clone-your-object-store-bucket)
+    - [4. Restart all pipeline and data loading ops](#restart-all-pipeline-and-data-loading-ops)
+    - [5. Deploy a 1.X Pachyderm cluster with cloned bucket](#deploy-a-1-x-pachyderm-cluster-with-cloned-bucket)
+    - [6. Restore the new 1.Y Pachyderm cluster from your backup](#restore-the-new-1-y-pachyderm-cluster-from-your-backup)
+    - [7. Load transactional data from checkpoint into new cluster](#load-transactional-data-from-checkpoint-into-new-cluster)
+    - [8. Disable the old cluster](#disable-the-old-cluster)
+    - [9. Reconfigure new cluster as necessary](#reconfigure-new-cluster-as-necessary)
+
+## Introduction
+As new versions of Pachyderm are released, you may need to update your cluster to get access to bug fixes and new features. 
+These updates fall into two categories, upgrades and migrations.
+
+An upgrade is moving between point releases within the same major release, 
+like 1.7.2 to 1.7.3.
+Upgrades are typically a simple process that require little to no downtime.
+They're covered in a [separate document](./upgrades.html).
+
+Migrations involve moving between major releases, 
+like 1.8.6 to 1.9.0.
+Migration is covered in this document. 
+
+In general, 
+Pachyderm stores all of its state in two places: 
+`etcd` 
+(which in turn stores its state in one or more persistent volumes,
+which were created when the Pachyderm cluster was deployed) 
+and an object store bucket 
+(something like AWS S3, MinIO, or Azure Blob Storage).
+
+In a migration, 
+the data structures stored in those locations need to be read, transformed, and rewritten, 
+so the process involves:
+
+1. bringing up a new Pachyderm cluster adjacent to the old pachyderm cluster
+1. exporting the old Pachdyerm cluster's repos, pipelines, and input commits
+1. importing the old cluster's repos, commits, and pipelines into the new
+   cluster, and then re-running all of the old cluster's pipelines from scratch
```

Comment:
This was a section from the old version that I didn't test, because I thought it was still accurate. From what I remember, when did the restore, the pipelines ran again themselves starting against  the head commit.  What's not clear here is there's no user action that needs to be taken; this is done automatically. I will remove it for clarity.